lib/logger/lib/logger/formatter.ex

```diff
@@ -253,6 +253,7 @@ defmodule Logger.Formatter do
   defp metadata(_, integer) when is_integer(integer), do: Integer.to_string(integer)
   defp metadata(_, float) when is_float(float), do: Float.to_string(float)
   defp metadata(_, pid) when is_pid(pid), do: :erlang.pid_to_list(pid)
+  defp metadata(_, port) when is_port(port), do: :erlang.port_to_list(port)
```

Comment:
Generally speaking we remove the prefixes to avoid extra data on loggers (see the `is_reference` clause below). Should we do the same here?